### PR TITLE
Use Pipfile.lock and provide update instructions

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/profiles/pull/100 to go through

### CODE REVIEW

File: Dockerfile.venv [hunk 1]
```diff
@@ -10,4 +13,4 @@ RUN virtualenv venv
 ENV VIRTUAL_ENV=${PROJECT_FOLDER}/venv
 
 COPY Pipfile Pipfile.lock ./
-RUN pipenv install ${pipenv_dev_arg} --deploy
+RUN pipenv sync ${pipenv_dev_arg}
```
Comment (de-code) [inline Dockerfile.venv]: Actually it's documented here: https://pipenv.readthedocs.io/en/latest/advanced/

> pipenv install --ignore-pipfile is nearly equivalent to pipenv sync, but pipenv sync will never attempt to re-lock your dependencies as it is considered an atomic operation. pipenv install by default does attempt to re-lock unless using the --deploy flag.

(The docs for `--ignore-pipfile` should really point to that)

Comment (giorgiosironi) [reply Dockerfile.venv]: Yes, `sync` seemed marginally better so I went for that

**Comment by giorgiosironi** [comment]
Some problems running the build because of Github downtime https://status.github.com/messages

**Comment by giorgiosironi** [comment]
Now it looks some tests are vulnerable to DST as the diffs have a 1-hour lag? What next, earthquake? Flood? Locusts?

**Comment by giorgiosironi** [comment]
Have to signoff now but I can continue cleaning up the project on Monday.

**Comment by giorgiosironi** [comment]
You wish... Tuesday.

**Comment by giorgiosironi** [comment]
Good fix, thank you

**Pull request merged**